Libraries/Image/Image.windows.js

```diff
@@ -113,6 +113,21 @@ var Image = React.createClass({
 
   statics: {
     resizeMode: ImageResizeMode,
+
+    getSize(
```

Comment:
While we're touching this file, can we do a full diff against Image.android.js / Image.windows.js and see if there are maybe any other JS-only features?